samples/cellular/nrf_cloud_multi_service/overlay_coap.conf

```diff
@@ -72,4 +72,4 @@ CONFIG_LOCATION_DATA_DETAILS=y
 CONFIG_APPLICATION_THREAD_STACK_SIZE=4096
 CONFIG_TEMP_ALERT_LIMIT=24
 
-CONFIG_NRF_CLOUD_COAP_DOWNLOADS=y
+#CONFIG_NRF_CLOUD_COAP_DOWNLOADS=y
```

Comment:
The CoAP Connector (proxy) has a white-list of domains it allows proxying to, so it will reject proxying to ota-cdn.memfault.com. Disable CoAP for image downloads, so it's downloaded over HTTPS instead.